examples/github/GitHub.js

```diff
@@ -0,0 +1,30 @@
+const { Reshuffle } = require('reshuffle')
+const { GitHubConnector } = require('reshuffle-github-connector')
+
```

Comment:
Need to document the example
e.g.

// This example uses reshuffle-github-connector
// Code and full documentation available on Github: https://github.com/reshufflehq/reshuffle-github-connector

// Get a Github token:
// - TODO Go to https://app.asana.com/0/developer-console
// - TODO  Under 'Personal Access Token'' > '+ New Access Token'
// - TODO Name it, then copy your token

Reply:
See https://github.com/reshufflehq/reshuffle/pull/460/files
for more examples